src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -3216,7 +3216,13 @@ func LaunchCustomQemu(ctx context.Context, path string, params []string, fds []*
 	/* #nosec */
 	cmd := exec.CommandContext(ctx, path, params...)
 	if len(fds) > 0 {
-		logger.Infof("Adding extra file %v", fds)
+		logger.Infof("Adding extra files %v", func() []string {
+			names := make([]string, len(fds))
+			for i, fd := range fds {
+				names[i] = fd.Name()
+			}
+			return names
+		}())
```

Comment:
Duplicated codes found in both `qemu.go` and `stratovirt.go` in this commit.
Though I know you add a new method to reduce the dup code in other commit, which looks enough.
We should keep commits clean and reduce unnecessary changes, so I suggest removing this commit.

Reply:
Thanks, I squashed two commits into a single commit.